### PR TITLE
Mobile and Desktop view of Tip calculator

### PR DESCRIPTION
**Pull request by sahibkaur**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -3,9 +3,12 @@ body {
...
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    height: 100vh;
 }
-
-
 .input {
     position: relative;
 }
```
Comment (divyanshugarg36) [inline style.css]: Class name input with a single property?
The naming convention can be used to specify what input you want to apply this relative property to.
We do have an input selector.

File: style.css [hunk 2]
```diff
@@ -15,7 +18,13 @@ input {
     border-radius: 10px;
```
Comment (divyanshugarg36) [inline style.css]: marking here,
I checked the frontend and found out the `width` property is used to reverse the effects of padding applied to the same input
This is not the recommended

Take a look at box-sizing property for a better and reliable solution.

File: style.css [hunk 3]
```diff
@@ -76,13 +91,12 @@ label {
...
 }
 
-.reset-btn{
-    background-color: hsl(172, 67%, 45%);
+#reset{
+
     padding: 10px;
     border-radius: 5px;
     border: none;
     width: 90%;
```
Comment (divyanshugarg36) [inline style.css]: The usage of padding and margin is better

Comment (divyanshugarg36) [reply style.css]: Same for this 
```
.price {
    width: 90%;
}
```

File: style.css [hunk 4]
```diff
@@ -127,4 +141,50 @@ label {
...
+    .left-container {
+        width: 45%;
+        display: flex;
+        flex-direction: column;
+        height: 95%;
+        justify-content: space-between;
+    }
+    button {
+        width: 100px;
+    }
```
Comment (divyanshugarg36) [inline style.css]: This applies CSS to all buttons 
Not a bad practice but it is recommended to have a generic CSS that can be applied to every button throughout the project

**Review by divyanshugarg36 — COMMENTED** [review]
Some overall improvements i noticed
1. Categorize classes
2. Duplicate class names
3. Less usage of margin and padding, and width is used as a replacement (Not recommended)
4. Code formatting (Use a formatter)

File: style.css (hunk 2 above)
Comment (sahibkaur) [reply style.css]: Even if I remove the width property, and use display:block; the input still doesn't take up the whole width.Screenshot 2023-06-19 at 7 08 26 PM

I searched on the internet and it says, I'll have to specify - width:100%.
https://stackoverflow.com/questions/35059569/html-input-element-not-taking-100-width

Comment (divyanshugarg36) [reply style.css]: Point is not to remove width, Poitn si to use 100% and then manage spacing etc using margin and padding

Comment (divyanshugarg36) [reply style.css]: Because 100% is 100% and 95% varies a lot in different screens

File: style.css (hunk 1 above)
Comment (sahibkaur) [reply style.css]: https://github.com/sahibkaur/tip-calculator/pull/3/commits/2a9b11befd879239eb4582fc6bcc8fe6e6f60da9

**Pull request merged**